rollout/replicaset.go

```diff
@@ -120,10 +119,53 @@ func (c *rolloutContext) reconcileNewReplicaSet() (bool, error) {
 	if err != nil {
 		return false, err
 	}
+
+	abortScaleDownDelaySeconds := time.Duration(defaults.GetAbortScaleDownDelaySecondsOrDefault(c.rollout))
+	if c.isScaleDownOnabort() {
+		c.log.Info("scale down on abort for the newRS")
```

Comment:
Would it be helpful to add the newRS being scaled down to the log line?

Reply:
log message is updated. Thanks.